versions/1.1.0-dev.md

```diff
@@ -141,6 +142,25 @@ The properties of the resolved `copy` object MUST be compatible with the target
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+### Parameter Object
+
+The object represents parameters that can be used with actions.
+
+#### Fixed Fields
+
+| Field Name | Type | Description |
+| ---- | :----: | ---- |
+| <a name="parameter-name"></a>name | `string` | **REQUIRED** A name for the parameter that is unique for the action the parameter is defined. The name MUST match the following regular expression `[a-zA-Z-0-9]+`. |
+| <a name="parameter-source"></a>source | `inline` or `environment` | **REQUIRED** Defines where the values for the parameters should be retrieved from. When `inline` is set, values are retrieved from the `values` property. When `environment` is set, the values are retrieved by the tool from environment variables. |
```

Comment:
I'm wondering here whether their is merit in pulling environment variable up a level, into a parameter "type" with a set of semantics. This feels like a worthy exercise in terms of consistency across implementations in tooling i.e. provide guidance to tooling makers on env variable handling, etc. Not critical for this feature I'm sure, but might be useful as a building block for the future.